app/services/booking_request_creator.rb

```diff
@@ -3,11 +3,16 @@ def create!(prisoner_step, visitors_step, slots_step)
     params = build_params(prisoner_step, visitors_step, slots_step)
     Visit.create!(params).tap { |visit|
       PrisonMailer.request_received(visit).deliver_later
+      add_log_metadata visit_id: visit.id
     }
   end
 
 private
 
+  def add_log_metadata(hash)
+    LogStasher.request_context.merge!(hash)
+  end
```

Comment:
I'd like to see a couple of unit tests in the spec to give this and the `id` parameters passed by the doubles a bit of context, please.  If I didn't know the history, I would not find it immediately clear why we were logging something that would potentially be in the database.